src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -125,6 +125,47 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
     }
   }
 
+  "Group Add Admins" should {
+    "Fail for nonexistent GroupId" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val maybeAddAdminsResult = sdk.groupAddAdmins(GroupId("tony"), Nil).attempt.unsafeRunSync
+      maybeAddAdminsResult.isLeft shouldBe true
+    }
+    "Succeed in the call, but fail to add an existing member" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val addAdminsResult = sdk.groupAddAdmins(validGroupId, List(primaryTestUserId)).attempt.unsafeRunSync.value
+      addAdminsResult.failed.length shouldBe 1
+      addAdminsResult.failed.head.error should include("User was already an admin")
+    }
+    "Fail for nonexistent UserId" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val maybeAddAdminsResult = sdk.groupAddAdmins(validGroupId, List(UserId("steve"))).attempt.unsafeRunSync
+      maybeAddAdminsResult.isLeft shouldBe true
+    }
+  }
+
+  "Group Remove Admins" should {
+    "fail for nonexistent GroupId" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val maybeRemoveAdminsResult = sdk.groupRemoveAdmins(GroupId("notarealgroup"), Nil).attempt.unsafeRunSync
+      maybeRemoveAdminsResult.isLeft shouldBe true
+    }
+    "Succeed in the call, but fail to remove a nonexistent UserId" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val removeAdminsResult =
+        sdk.groupRemoveAdmins(validGroupId, List(UserId("tony"))).attempt.unsafeRunSync.value
+      removeAdminsResult.failed.length shouldBe 1
+      removeAdminsResult.failed.head.error should include("could not be removed")
+    }
+    "fail to remove sole group admin" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val removeAdminsResult =
+        sdk.groupRemoveAdmins(validGroupId, List(primaryTestUserId)).attempt.unsafeRunSync.value
+      removeAdminsResult.failed.length shouldBe 1
+      removeAdminsResult.failed.head.error should include("could not be removed")
+    }
+  }
+
```

Comment:
According to #1 we are trying to move away from integration tests for the ironoxide-scala. If that's true, do we want these tests?

Can we just test the translation of `GroupAccessEditResult`? Or maybe we do still want integration tests? If so, we should rework/delete #1

Reply:
Yes, as far as I know that's still our goal. I mainly wrote these for my own sake as I'm not 100% confident in this repo yet. And I'm still unsure how to form the types of tests #1 calls for. But I'm definitely willing to take them out/change them.

Reply:
I'm fine with leaving them until we work on #1.